sway/tree/view.c

```diff
@@ -136,22 +151,33 @@ void view_map(struct sway_view *view, struct wlr_surface *wlr_surface) {
 	view->surface = wlr_surface;
 	view->swayc = cont;
 
+	view_init_subsurfaces(view, wlr_surface);
+	wl_signal_add(&wlr_surface->events.new_subsurface,
+		&view->surface_new_subsurface);
+	view->surface_new_subsurface.notify = view_handle_surface_new_subsurface;
+
 	arrange_windows(cont->parent, -1, -1);
 	input_manager_set_focus(input_manager, cont);
 
 	view_damage_whole(view);
 	view_update_outputs(view, NULL);
+
+	// TODO: create view children for subsurfaces
```

Comment:
:<

Reply:
Removed that outdated TODO. It's fixed by `view_init_subsurfaces` a few lines above.